### PR TITLE
Bump otel-collector to v0.39.0

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: exporter/skywalkingexporter/go.mod [hunk 1]
```diff
@@ -48,4 +41,11 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
 
+require (
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.38.0
```
Comment (djaglowski) [inline exporter/skywalkingexporter/go.mod:45]: 39?

File: receiver/postgresqlreceiver/go.mod [hunk 2]
```diff
@@ -4,31 +4,32 @@ go 1.17
 
 require (
 	github.com/stretchr/testify v1.7.0
-	go.opentelemetry.io/collector v0.38.1-0.20211103215828-cffbecb2ac9e
+	go.opentelemetry.io/collector v0.39.0
 	go.uber.org/multierr v1.7.0
 )
 
+require go.opentelemetry.io/collector/model v0.39.0
```
Comment (djaglowski) [inline receiver/postgresqlreceiver/go.mod:11]:
```suggestion
```

File: exporter/skywalkingexporter/go.mod (hunk 1 above)
Comment (bogdandrutu) [reply exporter/skywalkingexporter/go.mod:45]: this is contrib not core.

File: receiver/postgresqlreceiver/go.mod (hunk 2 above)
Comment (bogdandrutu) [reply receiver/postgresqlreceiver/go.mod:11]: Why?

Comment (djaglowski) [reply receiver/postgresqlreceiver/go.mod:11]: It's probably ok to leave, but it wasn't there before this PR. Component is partially added and not using this yet.

**Review by djaglowski — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Pull request merged**